src/qt/walletmodel.cpp

```diff
@@ -188,7 +188,7 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
             ++nAddresses;
 
             CScript scriptPubKey = GetScriptForDestination(DecodeDestination(rcp.address.toStdString()));
-            CRecipient recipient = {scriptPubKey, rcp.amount, rcp.fSubtractFeeFromAmount};
+            CRecipient recipient{CNoDestination{scriptPubKey}, rcp.amount, rcp.fSubtractFeeFromAmount};
```

Comment:
We know the destination here. Otherwise we wouldn't be able to decode it from the string. This should be:
```c++
CTxDestination dest = DecodeDestination(rcp.address.toStdString());
CRecipient recipient = {dest, rcp.amount, rcp.fSubtractFeeFromAmount};
```